src/index.html

```diff
@@ -1,17 +1,196 @@
+            type="text"
+            name="name"
+            id="name"
+            tabindex="2"
+            maxlength="12"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="how-old">How old are You?</label>
+          <input
+            type="number"
+            name="how-old"
+            id="how-old"
+            value="12"
+            min ="12"
+            max="100"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="birth-date">Full date of birth:</label>
+          <input
+            type="date"
+            name="birth-date"
+            id="birth-date"
+            tabindex="4"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="accept-agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="accept-agreement"
+            id="accept-agreement"
+            tabindex="5"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>Registration</legend>
+
+        <div class="form__input">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            tabindex="6"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="pass"
+            id="pass"
+            tabindex="7"
+            minlength="5"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__input">
+          Do you love cats?
+          <input
+            type="radio"
+            name="lovecat"
+            id="lovecat1"
+            tabindex="8"
+          >
+          <label for="lovecat1">Yes</label>
+          <input
+            type="radio"
+            name="lovecat"
+            id="lovecat2"
+          >
+          <label for="lovecat2">No</label>
+        </div>
+
+        <div class="form__input">
+          <label for="favorite-color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="favorite-color"
+            id="favorite-color"
+            tabindex="9"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="time-bed">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time-bed"
+            id="time-bed"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="favorite-car">What are your favorite brand of cars?</label>
+          <select name="favorite-car"
+            id="favorite-car"
+            multiple tabindex="10"
+          >
```

Comment:
Even the firs attribute should be on a new line
```suggestion
          <select 
          	name="favorite-car"
            id="favorite-car"
            multiple tabindex="10"
          >
```